astropy/coordinates/representation/spherical.py

```diff
@@ -540,6 +540,16 @@ def represent_as(self, other_class, differential_class=None):
                     lon=self.lon, lat=self.lat, differentials=diffs, copy=False
                 )
 
+            elif issubclass(other_class, RadialRepresentation):
```

Comment:
Unrelated? But good to add.

Reply:
It is the first commit - it is needed to ensure that the mask is kept correctly when transforming to `Radial`. Obviously, of very little use generally...